modules/ml/src/test/java/org/apache/ignite/ml/environment/LearningEnvironmentBuilderTest.java

```diff
@@ -39,7 +39,7 @@
     /** */
     @Test
     public void basic() {
-        LearningEnvironment env = LearningEnvironment.DEFAULT;
+        LearningEnvironment env = LearningEnvironment.DEFAULT_TRAINER_ENV;
```

Comment:
Why did you rename this object? I think it is not necessary

Reply:
This is exactly default learning environment for trainer, in contrast to workers, so I renamed it for clarity.